openshift/templates/postgresql/postgresql-deploy.dev.param

```diff
@@ -0,0 +1,23 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/postgresql/postgresql-deploy.yaml
+#=========================================================
+# NAME=postgresql
```

Comment:
For consistency with the naming convention we used in `email-verification`, we could update this to `iiw-book-db` (applies to other environments/param files as well).